Formula/docker-credential-magic.rb

```diff
@@ -0,0 +1,63 @@
+class DockerCredentialMagic < Formula
+  desc "Magic shim for Docker credential helpers"
+  homepage "https://github.com/docker-credential-magic/docker-credential-magic"
+  license any_of: ["Apache-2.0", "CC-BY-4.0"]
+
+  depends_on "go" => :optional
+
+  on_macos do
+    if Hardware::CPU.arm?
+      url "https://github.com/docker-credential-magic/docker-credential-magic/releases/download/v0.8.0/docker-credential-magic_Darwin_arm64.tar.gz"
```

Comment:
We don't install pre-built binaries in Homebrew/core. These should be built from source.

Reply:
thank you.